src/interface/src/app/plan/create-scenarios/create-scenarios.component.html

```diff
@@ -1,12 +1,70 @@
-<div class="create-scenarios-panel mat-elevation-z2" [@expandCollapsePanel]="panelExpanded ? 'expanded' : 'collapsed'">
-  <div class="create-scenarios-panel-content" [@expandCollapsePanelContent]="panelExpanded ? 'opaque' : 'transparent'">
-    <app-create-scenarios-intro *ngIf="planningStep === PlanStep.CreateScenarios"></app-create-scenarios-intro>
-    <app-set-priorities
-      *ngIf="planningStep === PlanStep.SetPriorities" (changeConditionEvent)="changeConditionEvent.emit($event)"
-      (changePrioritiesEvent)="changePrioritiesEvent.emit($event)" [plan$]="plan$"></app-set-priorities>
-    <app-constraints-panel *ngIf="planningStep === PlanStep.SetConstraints" (changeConstraintsEvent)="changeConstraintsEvent.emit($event)"></app-constraints-panel>
+<div class="create-scenarios-panel mat-elevation-z2"
+  [@expandCollapsePanel]="panelExpanded ? 'expanded' : 'collapsed'">
+  <div class="create-scenarios-panel-content"
+    [@expandCollapsePanelContent]="panelExpanded ? 'opaque' : 'transparent'">
+
+    <mat-stepper orientation="vertical" class="stepper"
+      (selectionChange)="selectedStepChanged($event)">
+
+      <!-- Step 1: Select condition score -->
+      <mat-step #step1>
+        <ng-template matStepLabel>
+          <div class="step-label-header">
+            Select condition score
+          </div>
+        </ng-template>
+        <app-create-scenarios-intro></app-create-scenarios-intro>
+      </mat-step>
+
+      <!-- Step 2: Set constraints -->
+      <mat-step #step2>
+        <ng-template matStepLabel>
+          <div class="step-label-header">Set constraints</div>
+          <div class="step-label-description" *ngIf="!stepStates[1].opened">
+            Constraints will be used to inform project areas and scenario options.
+          </div>
+        </ng-template>
+        <app-constraints-panel (changeConstraintsEvent)="changeConstraintsEvent.emit($event)">
+        </app-constraints-panel>
+      </mat-step>
+
+      <!-- Step 3: Select priorities -->
+      <mat-step>
+        <ng-template matStepLabel>
+          <div class="step-label-header">Select priorities</div>
+          <div class="step-label-description" *ngIf="!stepStates[2].opened">
+            Data from selected priorities inform optimization of project areas for desired impact.
+          </div>
+          <div class="step-label-description" *ngIf="stepStates[2].complete">
+            XX priorities selected
```

Comment:
Nit: remove XX?  If this is a placeholder for now, it's fine.

Reply:
Yep, it's a placeholder!